src/ioc/index.ts

```diff
@@ -1,12 +1,15 @@
 import IoCContainer from 'ioc-lite';
 
-// import { Logger } from '../services/logger';
-// import { HTTP } from '../services/http';
-// import { Users } from '../services/users';
+import { Logger } from '../services/logger';
+import { HTTP } from '../services/http';
+import { Users } from '../services/users';
 
-export const createIoCContainer = () =>  {
+export const createIoCContainer = () => {
   const ioc = new IoCContainer();
```

Comment:
Could you add a generic type to IoCContainer to allow TS to check the type of resource?